stub-idp/src/test/java/uk/gov/ida/stub/idp/resources/SingleIdpPromptResourceTest.java

```diff
@@ -0,0 +1,109 @@
+package uk.gov.ida.stub.idp.resources;
+
+import com.squarespace.jersey2.guice.JerseyGuiceUtils;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+import uk.gov.ida.saml.core.test.TestEntityIds;
+import uk.gov.ida.stub.idp.configuration.SingleIdpConfiguration;
+import uk.gov.ida.stub.idp.domain.Service;
+import uk.gov.ida.stub.idp.exceptions.FeatureNotEnabledException;
+import uk.gov.ida.stub.idp.repositories.Idp;
+import uk.gov.ida.stub.idp.repositories.IdpStubsRepository;
+import uk.gov.ida.stub.idp.resources.idp.SingleIdpPromptPageResource;
+import uk.gov.ida.stub.idp.services.ServiceListService;
+import uk.gov.ida.stub.idp.views.SingleIdpPromptPageView;
+
+import javax.ws.rs.core.Response;
+import java.net.URI;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Optional;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+import static org.mockito.Mockito.when;
+
+@RunWith(MockitoJUnitRunner.class)
+public class SingleIdpPromptResourceTest {
+
+    private final URI verifySubmissionUri = URI.create("http://localhost/initiate-single-idp");
```

Comment:
this could reflect the real endpoint (`/initiate-single-idp-journey`)

Reply:
Fair point ... sorted!